### PR TITLE
Observed aimpoints monitor

### PR DESCRIPTION
**Pull request by taldcroft**
Add script to monitor observed aimpoints and compute offsets from expected

### CODE REVIEW

**Comment by taldcroft** [comment]
Because this downloads event data from the archive it now needs to be run as a user with privileges.

File: update_observed_aimpoints.py [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+    Get the dmcoords-computed chipx and chipy for given event file
+    header keyword params.  NOTE: the ``dy`` and ``dz`` inputs
+    to dmcoords are flipped in sign from the ASOL values.  Generally the
+    ASOL DY/DZ are positive and dmcoord input values are negative.  This
+    sign flip is handled *here*, so input to this is ASOL DY/DZ.
+
+    :param keys: dict of event file keywords
+    """
+    # See the absolute_pointing_uncertainty notebook in this repo for the
+    # detailed derivation of this -15.5, 6.0 arcsec offset factor.  See the
```
Comment (jeanconn) [inline update_observed_aimpoints.py]: Where is the -15.5, 6.0 offset used?

Comment (taldcroft) [inline update_observed_aimpoints.py]: Sorry, that is a leftover unrelated comment.  I'll remove it in master.

Comment (taldcroft) [inline update_observed_aimpoints.py]: See d67f67c.

**Pull request merged**